lib/spack/spack/build_environment.py

```diff
@@ -656,6 +690,11 @@ def setup_package(pkg, dirty):
         dpkg.setup_dependent_package(pkg.module, spec)
         dpkg.setup_dependent_environment(spack_env, run_env, spec)
 
+    if (not dirty) and spack_env.variable_is_possibly_set('CPATH'):
```

Comment:
I'm not thrilled with the name of this function. Can we change it to `spack_env.is_unset(varname)` and reverse the boolean check?

Reply:
The name is now inverted